mono/metadata/reflection.c

```diff
@@ -2192,6 +2192,8 @@ mono_reflection_bind_generic_parameters (MonoReflectionType *type, int type_argc
 	if (mono_is_sre_type_builder (mono_object_class (type))) {
 		is_dynamic = TRUE;
 	} else if (mono_is_sre_generic_instance (mono_object_class (type))) {
+		/* Does this ever make sense?  what does instantiating a generic instance even mean? */
```

Comment:
Could this happen if you tried to do something like instantiating a partially-instantiated generic type? That ought to be disallowed by the SRE API, anyway.